### ISSUE TITLE
Save a screenshot of the current layout

### ISSUE THREAD

**Issue by marcosgriselli** [marcosgriselli/Sizes#22]
We could add a 'take screenshot' button that generates a `UIImage` from the `containedView` and displays the share sheet so we can save/share the image of the current layout in full size. 

Ideally, we can let users select if they want to place a label with the current device/orientation/font size on the generated image though this is not a priority.

Current situation when taking a screenshot when using Sizes on an iPad testing small phones layouts:
<img src="https://user-images.githubusercontent.com/14804033/47268911-2ec97780-d52d-11e8-9f0a-9995cc6d2158.png" width="500px"/>

**Comment by gtrunso** [comment]
I'm taking care of it

**Comment by marcosgriselli** [comment]
Great @gtrunso, I'll assign you the issue 🙂

**Comment by csknns** [comment]
I have implemented this! Did not see @gtrunso comment, sorry. 
![changesizesscreenshot](https://user-images.githubusercontent.com/10131788/47290412-06fb1200-d608-11e8-893d-7847b754da6e.gif)
Would you like a pull request to review it, or should I scrap it?

**Comment by csknns** [comment]
I have implemented this. Github has a few technical problems and did not allow me to fork the repo or add comments. Should I submit a PR or scrap my branch?

![changesizesscreenshot](https://user-images.githubusercontent.com/10131788/47290644-20e92480-d609-11e8-9af9-054c23e0ec8e.gif)

**Comment by marcosgriselli** [comment]
Hey @csknns thanks for working on this! @gtrunso wasn't able to send the PR yesterday due to the GitHub issues. I think this is a great opportunity for both of you to work together since this functionality will require a bit of extra work to be perfect. Also, I recommend we wait until the issues with GitHub are solved since is messing up with the workflows.

**Comment by marcosgriselli** [comment]
I'd avoid pushing new branches or sending PRs. @csknns I just got an email from a PR of yours but it's not showing up on GitHub for me so let's wait until GitHub is working properly to work on this.

**Comment by csknns** [comment]
@marcosgriselli I agree, I will close the PR for now and wait until Github reports everything is back to normal.
It will be a great to collaborate with @gtrunso on this issue!

**Comment by jbuchman** [comment]
Please make sure that the screenshots are at the native resolution of the view as opposed to the resolution of the device. I had implemented something similar for my use case, but found that if I screenshot the SizesWindow or SizesViewController's views I ended up with a screenshot at the device's resolution instead of the resolution of the simulated size...
